src/components/Sidebar/index.tsx

```diff
@@ -39,37 +39,6 @@ const search = debounce((router: NextRouter, searchTerm: string) => {
     : router.push(pathname, as);
 }, 200);
 
-const aggregateMemberLinks = (
-  members: Member[],
-  field: keyof Member,
-  prefix: string
-) => {
-  const isArrayType = Array.isArray(members[0][field]);
-  let aggregateGroup: [fieldValue: string, group: Member[]][] = [];
-  if (isArrayType) {
-    let fieldAggregate: string[] = Array.from(
-      new Set(members.flatMap((member) => member[field]))
-    ) as string[];
-    aggregateGroup = fieldAggregate.map((fieldValue) => [
-      fieldValue,
-      members.filter((member) =>
-        (member[field] as string[]).includes(fieldValue)
-      ),
-    ]);
-  }
-
-  return (isArrayType
-    ? aggregateGroup
-    : Object.entries(groupBy(members, field))
-  )
-    .map(([fieldValue, group]) => ({
-      text: fieldValue,
-      count: (group as any)?.length,
-      href: `/${prefix}/${normalizeParam(fieldValue)}`,
-    }))
-    .filter(({ text }) => text);
-};
-
```

Comment:
Yikes. Ha, being able to query for this data makes the logic _tons_ easier. You can see what replaced this in `data/sidebar.ts`.

Reply:
Makes me think of... 

<img width="772" alt="Screen Shot 2020-11-15 at 10 48 50 AM" src="https://user-images.githubusercontent.com/236943/99193805-2c4ba300-2730-11eb-9437-de346beec2b4.png">

Wondering if we should do https://www.prisma.io/docs/concepts/overview/prisma-in-your-stack/graphql at some point?